### PR TITLE
Coffee data

### PR DESCRIPTION
**Pull request by DEJENERUTA**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,11 +1,9 @@
-import "./App.css";
-
 const App = () => {
-  return (
-    <div className="App">
-      <h1>Coffee guide</h1>
-    </div>
-  );
+	return (
```
Comment (bence-toth) [inline src/App.js:2]: Can you please update your editor settings to use spaces instead of tabs?

File: src/App.test.js [hunk 2]
```diff
@@ -1,8 +0,0 @@
-import { render, screen } from '@testing-library/react';
```
Comment (bence-toth) [inline src/App.test.js:1]: Can you please restore this file?

File: src/coffee.js [hunk 3]
```diff
@@ -0,0 +1,312 @@
+const coffee = [
```
Comment (bence-toth) [inline src/coffee.js:1]: Why is this in two files (`coffee.js` and `coffeeObject.js`)? Can we have it in only one file?

File: src/coffee.js [hunk 4]
```diff
@@ -0,0 +1,312 @@
...
+			"cappuccino",
+			"cortado",
+			"flat white",
+		],
+		origin: [
+			{
+				name: "Limu",
+				latitude: 9.16702,
+				longitude: 36.34015,
+				altitude: 1750 - 1950,
```
Comment (bence-toth) [inline src/coffee.js:19]: Can you use `altitudeMin` and `altitudeMax`?

File: src/reportWebVitals.js [hunk 5]
```diff
@@ -1,13 +0,0 @@
-const reportWebVitals = onPerfEntry => {
```
Comment (bence-toth) [inline src/reportWebVitals.js:1]: Can you restore this file please?

File: src/index.js [hunk 6]
```diff
@@ -1,17 +1,11 @@
-import React from 'react';
```
Comment (bence-toth) [inline src/index.js:1]: Can you revert the changes to this file please?

File: src/index.css [hunk 7]
```diff
@@ -1,11 +1,11 @@
 * {
```
Comment (bence-toth) [inline src/index.css:1]: Can you revert the changes to this file please?

File: src/coffeObject.js [hunk 8]
```diff
@@ -0,0 +1,246 @@
+const coffeeObject = [
+	{
+		name: "Aynu",
+		profile: "clean", // or "rich",
```
Comment (bence-toth) [inline src/coffeObject.js:4]: Can you remove the end-of-line comments?

File: src/coffeObject.js [hunk 9]
```diff
@@ -0,0 +1,246 @@
...
+		characteristic: "sweet", // or "acidic" or "round"
+		taste: ["chocolate", "spicy", "floral"],
+		processed: "Washed",
```
Comment (bence-toth) [inline src/coffeObject.js:7]: Let's use all lowercase, except for the names (coffee name, and origin name)

File: src/coffeObject.js [hunk 10]
```diff
@@ -0,0 +1,246 @@
...
+			},
+			{
+				name: "Limu Kosa",
+				latitude: 8.16638,
+				longitude: 37.17096,
+				altitudeMin: 1750,
+				altitudeMax: 1950,
+			},
+		],
+		name: "Beza",
```
Comment (bence-toth) [inline src/coffeObject.js:25]: It looks like you should have closed the previous object here and started a new one

File: src/coffee.js [hunk 11]
```diff
@@ -0,0 +1,312 @@
+const coffee = [
+	{
+		name: "Aynu",
+		tastingProfile: ["Rounded", "Fruit Acidity", "Nut Aroma"],
+		processing: ["Washed"],
+		roastProfile: ["Dark Roast, Medium Roast, Light Roast"],
```
Comment (bence-toth) [inline src/coffee.js:6]: This should be an array of strings:
```suggestion
		roastProfile: ["dark roast", "medium roast", "light roast"],
```

**Review by bence-toth — CHANGES_REQUESTED** [review]
Added some comments. Some mistakes were repeated, but I only commented in one place.

File: src/coffeObject.js (hunk 10 above)
Comment (DEJENERUTA) [reply src/coffeObject.js:25]: I didn't get what you mean, when I look at it, it seems I closed it at the right place

File: src/coffee.js (hunk 3 above)
Comment (DEJENERUTA) [reply src/coffee.js:1]: I thought they are two different data, coffee.js is all data of the coffees and coffeeObject.js is the data we will use for only making the questions, but I can see what you mean, I am little bit afraid to have it in one file, just I am thinking to not make a mess

Comment (bence-toth) [reply src/coffee.js:1]: I'm not sure what you mean. We need data about the coffee. We only need it in one place.

File: src/coffeObject.js (hunk 10 above)
Comment (bence-toth) [reply src/coffeObject.js:25]:
```js
{
  name: "Aynu",
  profile: "clean", // or "rich",
  characteristic: "sweet", // or "acidic" or "round"
  taste: ["chocolate", "spicy", "floral"],
  processed: "Washed",
  qGrade: 88,
  origin: [
    {
      name: "Limu",
      latitude: 9.1667,
      longitude: 36.34,
      altitudeMin: 1650,
      altitudeMax: 1950,
    },
    {
      name: "Limu Kosa",
      latitude: 8.16638,
      longitude: 37.17096,
      altitudeMin: 1750,
      altitudeMax: 1950,
    },
  ],
  name: "Beza",
  profile: "clean", // or "rich",
  characteristic: "sweet", // or "acidic" or "round"
  taste: ["chocolate", "spicy", "floral"],
  processed: "sundried",
  qGrade: 92,
  origin: [
    {
      name: "Yirgachefe",
      latitude: 6.16272,
      longitude: 38.20583,
      altitudeMin: 1900,
      altitudeMax: 2100,
    },
    {
      name: "Yirga Chefe",
      latitude: 6.334640683301095,
      longitude: 39.00524849697445,
      altitudeMin: 1700,
      altitudeMax: 1950,
    },
  ]
  // ...
}
```

vs.

```js
{
  name: "Aynu",
  profile: "clean", // or "rich",
  characteristic: "sweet", // or "acidic" or "round"
  taste: ["chocolate", "spicy", "floral"],
  processed: "Washed",
  qGrade: 88,
  origin: [
    {
      name: "Limu",
      latitude: 9.1667,
      longitude: 36.34,
      altitudeMin: 1650,
      altitudeMax: 1950,
    },
    {
      name: "Limu Kosa",
      latitude: 8.16638,
      longitude: 37.17096,
      altitudeMin: 1750,
      altitudeMax: 1950,
    },
  ],
}, /// <--- Close previous object here
{  /// <--- Open new object here
  name: "Beza",
  profile: "clean", // or "rich",
  characteristic: "sweet", // or "acidic" or "round"
  taste: ["chocolate", "spicy", "floral"],
  processed: "sundried",
  qGrade: 92,
  origin: [
    {
      name: "Yirgachefe",
      latitude: 6.16272,
      longitude: 38.20583,
      altitudeMin: 1900,
      altitudeMax: 2100,
    },
    {
      name: "Yirga Chefe",
      latitude: 6.334640683301095,
      longitude: 39.00524849697445,
      altitudeMin: 1700,
      altitudeMax: 1950,
    },
  ]
  // ...
}
```

**Comment by bence-toth** [comment]
I have unresolved some conversations. Please don't resolve any of them before you have confirmed the you pushed the changes that addresses them.

**Review by bence-toth — CHANGES_REQUESTED** [review]

**Comment by bence-toth** [comment]
Closing this in favour of #17
